addOns/scanpolicies/src/main/javahelp/help/contents/policy-dev-cicd.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Developer CI/CD Policy
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Developer CI/CD Policy</H1>
+
```

Comment:
Include a high level one liner? Something like "This policy is designed to be used by developers in a CI/CD pipeline" ?
Similar one lines for the other policies?